android/src/main/java/com/applovin/reactnative/AppLovinMAXNativeAdView.java

```diff
@@ -383,14 +383,29 @@ public void onNativeAdClicked(final MaxAd ad)
 
     private void sendAdLoadedReactNativeEventForAd(final MaxNativeAd ad)
     {
-        // 1. AdInfo for publisher to be notified via `onAdLoaded`
-
         WritableMap nativeAdInfo = Arguments.createMap();
         nativeAdInfo.putString( "title", ad.getTitle() );
         nativeAdInfo.putString( "advertiser", ad.getAdvertiser() );
         nativeAdInfo.putString( "body", ad.getBody() );
         nativeAdInfo.putString( "callToAction", ad.getCallToAction() );
 
+        MaxNativeAdImage icon = ad.getIcon();
+        if ( icon != null )
+        {
+            WritableMap iconObj = Arguments.createMap();
+
+            if ( icon.getUri() != null )
+            {
+                iconObj.putString( "url", icon.getUri().toString() );
+            }
+            else if ( icon.getDrawable() != null )
+            {
+                iconObj.putBoolean( "image", true );
+            }
```

Comment:
This is a bit confusing since `image` has `url` which is image data vs `image` which is whether or not it exists. What did the pub ask for? Can we just pass whether or not the icon image exists (url or drawable).

Reply:
I'm trying to combine two objects here since we send almost two identical native ad objects to the JS layer.  I need to revert the combining if it should be like `isIconViewAvailable`.    Is there another way to work for both the pub and us?

Reply:
When creating APIs (b/c it is very hard to revert once pushed), I try to start with as base of a case as possible. So in this case, I would just expose API `isIconImageAvailable` (not icon view, since that's technically belonging to the pub's integration and not our native ad).